pkgs/applications/kde/itinerary.nix

```diff
@@ -0,0 +1,62 @@
+{ mkDerivation
+, lib
+, extra-cmake-modules
+, karchive
+, kcalendarcore
+, kcontacts
+, kdbusaddons
+, kfilemetadata
+, kholidays
+, kio
+, kirigami-addons
+, kitemmodels
+, kitinerary
+, kmime
+, knotifications
+, kosmindoormap
+, kpkpass
+, kpublictransport
+, kunitconversion
+, libquotient
+, networkmanager-qt
+, qqc2-desktop-style
+, qtpositioning
+, qtquickcontrols2
+, shared-mime-info
+}:
+
+mkDerivation {
+  pname = "itinerary";
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    shared-mime-info # for update-mime-database
```

Comment:
Maybe poke upstream about removing this dependency? It doesn't seem to be used anyway.

Reply:
There is no direct usage of it but I am not sure if KDE is not using it somewhere in their cmake structure since I am not familiar with their infrastructure

Reply:
They shouldn't, there's nothing like that in most other packages or in ECM.